bin/dev_setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Install components, setup DB and Solr:
+bundle install
+bundle exec rake argo:install
+
+# Run the server
+bundle exec rake jetty:stop  # Ensure it's not running
+bundle exec rake jetty:start # This may take a few minutes
+
+# Load and index records in the 'development' Solr core
+bundle exec rake argo:repo:load
```

Comment:
is there a way to have `argo:repo:load` wait a bit or try to detect whether jetty's actually started?  `rake jetty:start` often returns before the server's accepting requests (i usually hit the web UI to make sure it's up before i run `repo:load`).